### ISSUE TITLE
move tabs to change tab order

### ISSUE THREAD

**Issue by CalvinT05** [acmutd/bsg#126]
User should be able to hold and drag the tabs so that they can change the tab order like LeetCode.

Not very important but is great for UX consistency.
